spec/models/case/sar/offender_spec.rb

```diff
@@ -226,6 +226,24 @@
       end
     end
   end
+  
+  describe '#request_dated' do
+    context 'invalid value' do
+      it 'errors' do
+        expect {
+          build(:offender_sar_case, request_dated: 'wibble')
+        }.to raise_error ArgumentError
+      end
+    end
+
+    context 'received date cannot be in future' do
```

Comment:
```suggestion
    context 'request_dated cannot be in future' do
```
maybe worth amending the label here